### ISSUE TITLE
v1.10.1 image unavailable

### ISSUE THREAD

**Issue by benjyiw** [zalando/postgres-operator#2411]
Hey there,

You guys recently updated your helm chart which now references an image tag "v1.10.1" but that image tag does not exist in your registry, "registry.opensource.zalan.do/acid/postgres-operator:v1.10.1" 

I noticed due to a k8s cluster not having the helm chart version pinned, so the operator ended up in an image pull backoff. No bug to report other than just that image being missing.

**Comment by ibotty** [comment]
It's available at least now from my clusters.

**Comment by FxKu** [comment]
Yes, it's available now. Every time I tag a release I need to trigger our CI pipeline to run e2e tests and push the tested image which takes around 45 minutes. And unfortunately, e2e tests can be flaky and sometimes fail certain tests so I have to restart the pipeline again.

I thought doing it in the night would save me from people creating issues about missing images right after the release, but here we are :smiley: @benjyiw I'm curious if the `image pull backoff`error happens when a new instance of the operator is spawned or when nodes are rotated? Want to make sure that running stuff does not break as soon as I hit the release button.

**Comment by benjyiw** [comment]
Sorry for the noise!

> I'm curious if the image pull backofferror happens when a new instance of the operator is spawned or when nodes are rotated?
I'm using ArgoCD to keep helm releases up to date. I didn't have the postgres-operator version pinned to a specific version of the chart, so a minute or two after the helm chart was updated, my release auto-updated to the new helm version. The deployment tried to roll out a new pod with the new image version and ended up in an image pull backoff, but that didn't break the existing pod on the previous version. I noticed only due to alerting that I have in the cluster for such things.

Also fwiw, I believe deployments by default don't remove the old replicaset until the new replicaset is considered ready, so I don't think a node rotation would have made the situation worse by any means because the old replicaset would replace the old pod with the same old version.

Anyways, thanks for the work on the operator!

I'll close this.
